src/mmwtex.c

```diff
@@ -1086,15 +1086,16 @@ vstring tokenToTex(vstring mtoken, long statemNum /* for error msgs */)
     } // Next i
 
     // Make all letters Roman in math mode
-    if (!g_htmlFlag)
+    if (!g_htmlFlag) { // LaTeX
       let(&tex, cat("\\mathrm{", tex, "}", NULL));
-  } // End if
+    }
+  } // if (texDefsPtr)
 
   return tex;
 } // tokenToTex
 
-// Converts a comment section in math mode to TeX.  Each math token
-// MUST be separated by white space.   TeX "$" does not surround the output.
+// Convert a comment section in math mode to LaTeX.  Each math token
```

Comment:
```suggestion
// Converts a comment section in math mode to LaTeX.  Each math token
```
I think indicative mood is more conventional for documentation and is already used elsewhere to my knowledge